modules/testingModule/index.js

```diff
@@ -0,0 +1,100 @@
+import { setLabels as setAnalyticLabels } from "../../libraries/analyticsAdapter/AnalyticsAdapter.js";
+import { ACTIVITY_ACCESS_USER_IDS, ACTIVITY_ENRICH_EIDS } from "../../src/activities/activities.js";
+import { MODULE_TYPE_UID } from "../../src/activities/modules.js";
+import { ACTIVITY_PARAM_COMPONENT_NAME, ACTIVITY_PARAM_COMPONENT_TYPE } from "../../src/activities/params.js";
+import { registerActivityControl } from "../../src/activities/rules.js";
+import { config } from "../../src/config.js";
+import { getCoreStorageManager } from "../../src/storageManager.js";
+import { deepEqual, logError, logInfo } from "../../src/utils.js";
+
+const MODULE_NAME = 'testingModule';
+export const suppressionMethod = {
+  SUBMODULES: 'submodules',
+  EIDS: 'eids'
+};
+
+export const storeSplitsMethod = {
+  PAGE: 'page',
+  SESSION: 'session',
+  LIFE_OF_USER: 'lifeOfUser'
+};
+
+let moduleConfig;
+let rules = [];
+
+export function init(storageManager = getCoreStorageManager('testingModule')) {
```

Comment:
using coreStorageManager means that the consent check is done only if one explicitly enables it with [`strictStorageEnforcement`](https://docs.prebid.org/dev-docs/modules/tcfControl.html). We probably don't want this - I'd expect this to follow the same rules as something like sharedId (which always needs publisher consent regardless of that flag).


To do this we need to either add a special case here 

https://github.com/prebid/Prebid.js/blob/66aab4d3a875b65ad25f0429ab201b8d2051a3f6/modules/tcfControl.js#L247-L248

or use a different module type from `MODULE_TYPE_PREBID` and register its gvl ID as `VENDORLESS_GVLID` here:

https://github.com/prebid/Prebid.js/blob/66aab4d3a875b65ad25f0429ab201b8d2051a3f6/src/consentHandler.js#L147